### PR TITLE
added chat sdk

### PR DESCRIPTION
**Pull request by euniceyoh**

### CODE REVIEW

**Comment by euniceyoh** [comment]
update: tried testing but couldn't get it to work. ready for review.  
~~not completely ready for review rn but wanted to share updates~~

**Comment by euniceyoh** [comment]
Test: 
Screen Shot 2022-04-17 at 6 10 59 PM

File: src/components/Chat.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import {App as SendBirdApp} from "sendbird-uikit";
+import "sendbird-uikit/dist/index.css";
+
+import styles from './chat.module.css'
+
+import React from "react";
+import {Navigation} from "./index";
+
+var YOUR_APP_ID = "45391E28-BD64-49C5-89A5-C14E1A5E5A0C"
```
Comment (JoshuaLawson8) [inline src/components/Chat.jsx]: should this be env values?

File: src/App.jsx [hunk 2]
```diff
@@ -21,11 +22,13 @@ function App() {
         path="/UserDashboard"
         element={<UserDashboard />}
```
Comment (JoshuaLawson8) [inline src/App.jsx:23]: should also pass user in as props here

**Review by JoshuaLawson8 — APPROVED** [review]
Looks good overall, I think that we need some restructuring of the user state

File: src/components/Chat.jsx (hunk 1 above)
Comment (euniceyoh) [reply src/components/Chat.jsx]: yeah, it should be changed to env now

File: src/App.jsx (hunk 2 above)
Comment (euniceyoh) [reply src/App.jsx:23]: i'm not implementing the user dashboard, but maybe bring this up to person responsible?

**Comment by euniceyoh** [comment]
issues w/ testing
Screen Shot 2022-04-18 at 6 37 48 PM

**Review by isabellapiz — COMMENTED** [review]
looks good to me :)

**Review by christinang3030 — APPROVED** [review]
Looks good! Worked for me.

**Comment by euniceyoh** [comment]
> looks good to me :)

thanks for reviewing! do u mind explicitly choosing approval in ur review so it shows as 3 approvals on the PR?

**Review by isabellapiz — APPROVED** [review]

**Pull request merged**